azure/container-registry/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "resource_group_name" {
+  type = string
+}
+
+variable "name" {
+  type = string
+}
+
+variable "region" {
+  type = string
+}
+
+variable "project" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "admin_enabled" {
+  type    = bool
+  default = true
+}
```

Comment:
As discussed, this is fine for now. Can you please add an issue to the backlog to come back to this and refactor this to use either  a service principal or a managed identity? Thanks!